src/components/itemPage/Item.jsx

```diff
@@ -89,8 +90,39 @@ class Item extends Component {
     );
   }
 
+  listenEventSource = async () => {
```

Comment:
In this function, there are two totally different parts (on a logical and time perspectives) that appears too "tangled":

- the first call to `_fetchItems` (which personally I would have let in its original place),
- the initialization of the event source (which requires to get the `update_seq` first).